thrall/app/lib/ThrallMessageConsumer.scala

```diff
@@ -43,7 +43,7 @@ object ThrallMessageConsumer extends MessageConsumer(
     withImageId(exports)(id => ElasticSearch.deleteImageExports(id))
 
   def updateImageUserMetadata(metadata: JsValue): Future[UpdateResponse] =
-    withImageId(metadata)(id => ElasticSearch.applyImageMetadataOverride(id, metadata \ "data"))
+    withImageId(metadata)(id => ElasticSearch.applyImageMetadataOverride(id, metadata))
```

Comment:
For consistency with other methods, would it make sense to still pass the data as second argument and the date as third? Avoids having to teach `ElasticSearch` about message structure, and keeps that in this message consumer instead?